bundles/org.openhab.binding.radiobrowser/src/main/java/org/openhab/binding/radiobrowser/internal/api/RadioBrowserApi.java

```diff
@@ -235,7 +235,8 @@ public void updateStations() throws ApiException {
 
     private @Nullable Station searchStationName(String name) throws ApiException {
         try {
-            String returnContent = sendGetRequest("/json/stations/byname/" + name);
+            String returnContent = sendGetRequest(
+                    "/json/stations/byname/" + URLEncoder.encode(name, "UTF-8").replace("+", "%20"));
```

Comment:
Are you sure? I would have expected the + to already be encoded as %2B

Reply:
Yes I am sure, have used this a lot in other bindings.
https://stackoverflow.com/questions/4737841/urlencoder-not-able-to-translate-space-character